CHANGELOG.md

```diff
@@ -1,5 +1,70 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.4.4
+
+<!-- generated comparing v5.4.3..master -->
+
+_Feb 28, 2022_
+
+A big thanks to the 17 contributors who made this release possible. Here are some highlights ✨:
+
+- ✨ New `Input` and `Sheet` components were added in the experimental joy design system by @hbjORbj (#31124, #31086) @hbjORbj
+- Several 🐛 bug fixes and 📚 documentation improvements
+
+### `@mui/material@5.4.4`
+
+- &#8203;<!-- 28 -->[Autocomplete] Have Autocomplete with multiline textfield log a warning instead of an error (#30680) @iclaude3
+- &#8203;<!-- 27 -->[Chip] Fix ellipsis when the children is too long (#31087) @PunitSoniME
+- &#8203;<!-- 14 -->[Input] Export InputBase's classes from the classes const (#31186) @mnajdova
+- &#8203;<!-- 29 -->[TextField] Fix Horizontal scroll when label too long (#31187) @RedHeadphone
+- &#8203;<!-- 08 -->[styles] Fix typo in import error (#31167) @davwheat
+
+### `@mui/system@5.4.4`
+
+- &#8203;<!-- 07 -->[system] Fix executing server-side Emotion component as function interpolation 2 (#31024) @Andarist
+- &#8203;<!-- 06 -->[system] Fix sx prop types when CSS variables are used with nested selectors (#31163) @mnajdova
+- &#8203;<!-- 05 -->[system] Fix `CssVarsProvider` theme mutation (#31148) @siriwatknp
+
+### `@mui/codemod@5.4.4`
```

Comment:
Not sure about this but ideally this should be `5.5.0` because of the new Codemod?

Reply:
Was thinking about the same thing, but then it would be different from the main package version, and I am not sure it's worth updating it to all packages, just because of this..

Reply:
Me too. I'm okay with the current change.